index.js

```diff
@@ -150,3 +170,10 @@ gontactForm.addEventListener('submit', (e) => {
     gontactForm.submit();
   }
 });
+
+// display changes
+dataForm.forEach((inputInfo) => {
+  name.value = inputInfo.name;
+  email.value = inputInfo.email;
+  msg.value = inputInfo.msg;
+});
```

Comment:
- [ ] To make it work, you need to get the data from the `localStorage` and display the data in the several form input. 👍 

#### Hint: `const obtainedData = JSON.parse(localStorage.getItem('info'));`. Then you assign the elements with the values from the `obtainedData.` 👍